src/app/objective/objective.component.css

```diff
@@ -9,3 +9,10 @@
 .assign-button-enabled a { color: #674ab7; text-decoration: none; }
 .assign-button-enabled a:hover { text-decoration: underline; }
 .assign-button-disabled { color: grey; }
+.sum-all { display: flex; justify-content: center;   width: 10%; align-items: center;} 
+.sum { font-weight: bold;}
+.objective {float: left; width: 90%;}
+.objective-all {display: flex;}
+.background-safe { color: grey; }
```

Comment:
This class doesn't affect the background, so shouldn't have `background` in the name.

The name should reflect the actual meaning we're going for; I'd suggest something like `resource-csum-ok`, `resource-csum-marginal`, and `resource-csum-excess` (open to better suggestions).